client/src/pages/Achieves/Achieve/Achieve.tsx

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+import { FC, useState } from "react";
+import { useTranslation } from "react-i18next";
+import cn from "classnames";
+import { ReactSVG } from "react-svg";
+import styles from "./Achieve.module.scss";
+import placeholder from "../../../assets/images/Achieves/unknown.svg";
+import { AchieveInterface } from "../types";
+import closeIcon from "../../../assets/images/Achieves/close-icon.svg";
+
+interface AchieveProps {
+  achieve: AchieveInterface;
+  isAchieveOpen: boolean;
+}
+
+export const Achieve: FC<AchieveProps> = ({ achieve, isAchieveOpen }) => {
+  const { t } = useTranslation();
+  const [isPopupOpen, setIsPopupOpen] = useState(false);
+  const toggleDescription = () => {
+    setIsPopupOpen(!isPopupOpen);
```

Comment:
I would suggest you rewrite this like (prev) => !prev. It is better to use the previous value in such cases.